services/viva-ms/helpers/formHelpers.js

```diff
@@ -12,6 +12,11 @@ function getAttributeFromAnswerFieldId(fieldId) {
   return attribute;
 }
 
+function getSecondStringFromDotNotatedString(sourceString) {
```

Comment:
If this function is to live in the form-helpers lib. It should be renamed to something more accurate and repurposed.

```JS
getAttributeFromDotNotation(string, position) {
  const attributes = string.split('.');
    return attributes[position];
}
```